### PR TITLE
Fix Typing Error

### LINKED ISSUE #6549: CMS Tutorial - Error when updating beforeSave method
When I copy and paste the `beforeSave` method from the [Persisting the Tag String](https://book.cakephp.org/4/en/tutorials-and-examples/cms/tags-and-users.html#persisting-the-tag-string) section I get a type error. I believe this is because we don't import `Cake\Event\EventInterface` in our initial implementation of the `beforeSave` method in the [Adding Simple Slug Generation](https://book.cakephp.org/4/en/tutorials-and-examples/cms/articles-controller.html#adding-simple-slug-generation) section.

Error output is:
```
Argument 1 passed to App\Model\Table\ArticlesTable::beforeSave() must be an instance of App\Model\Table\EventInterface, instance of Cake\Event\Event given, called in /srv/vendor/cakephp/cakephp/src/Event/EventManager.php on line 310
```
---

I will submit a PR to remove the typing from the method params to match the [Adding Simple Slug Generation](https://book.cakephp.org/4/en/tutorials-and-examples/cms/articles-controller.html#adding-simple-slug-generation) section.

### PR DESCRIPTION
**Pull request by amayer5125**
Remove EventInterface type from beforeSave method example
Fixes #6549

### CODE REVIEW

**Comment by othercorey** [comment]
The correct fix is to add the `use` statement for `EventInterface`.

**Comment by amayer5125** [comment]
Okay, I will update the [Adding Simple Slug Generation](https://book.cakephp.org/4/en/tutorials-and-examples/cms/articles-controller.html#adding-simple-slug-generation) section to include the `use` statement.

**Comment by amayer5125** [comment]
@othercorey I updated this PR with the requested changes.

File: en/tutorials-and-examples/cms/articles-controller.rst [hunk 1]
```diff
@@ -319,10 +319,12 @@ typically a URL-safe version of an article's title. We can use the
     use Cake\ORM\Table;
     // the Text class
     use Cake\Utility\Text;
+    // the EventInterface class
+    use Cake\Event\EventInterface
```
Comment (othercorey) [inline en/tutorials-and-examples/cms/articles-controller.rst]:
```suggestion
    use Cake\Event\EventInterface;
```

Comment (amayer5125) [reply en/tutorials-and-examples/cms/articles-controller.rst]: :man_facepalming: Fixed.

**Comment by othercorey** [comment]
Thanks!

**Pull request merged**